client/templates/articles.coffee

```diff
@@ -11,13 +11,46 @@ Template.articles.helpers
   diseases: ->
     Template.instance().diseases.get()
 
+  articlesReady: ->
+    Template.instance().articlesReady.get()
+
 Template.articles.events
-  "change #diseases": (event) ->
+
+  "click .block": (event, template) ->
+    $(".block").removeClass("current")
+    $(event.target).addClass("current")
+
+  "change #diseases": (event, template) =>
+    $(".block").removeClass("current")
+    $(".month").addClass("current")
```

Comment:
you could chain ln24 and ln25 like this: $(".block").removeClass("current").find(".month").addClass("current");
Things like that make interfaces a bit more snappy.